apps/wavelet/CMakeLists.txt

```diff
@@ -10,60 +10,28 @@
 project(wavelet)
 cmake_minimum_required(VERSION 3.1.3)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
+# halide.cmake requires that HALIDE_DISTRIB_DIR be set to point to the Halide distribution folder we use.
```

Comment:
Per conversation with Steven, this is a standalone example and thus somewhat different from the other apps. We should make sure this is documented here and perhaps also make sure there's a pointer in the top-level cmake doc file if this is particularly useful for folks to look at.

Reply:
IMHO we should rename this to (e.g.) "apps/standalone_example" (and possible move the 'bazeldemo' build files into it as well).

(If I had my druthers, I'd vote for making *all* of the apps 'standalone', with the requirement that 'make distrib' be done first, but that's a separate debate.)